sdk/appconfiguration/test-resources.json

```diff
@@ -48,15 +48,15 @@
             "defaultValue": "azconfig-net",
             "type": "string"
         },
-        "endpointSuffix": {
+        "azConfigEndpointSuffix": {
```

Comment:
Where do we get this value from?

Reply:
This gets overridden externally by the New-TestResources.ps1 script. These suffixes are stored in cloud-specific config files (in keyvault, since they mostly contain secrets) that get referenced by the pipeline: https://github.com/Azure/azure-sdk-for-net/blob/master/eng/pipelines/templates/stages/archetype-sdk-tests.yml#L51-L52

Reply:
Are the suffixes themselves secret?

Reply:
They are not. @weshaggard and I have chatted about having a separate, non-secret config for all these values too. Originally we put them in the existing config for convenience.

Reply:
I'm asking because there is already some suffix handling logic in New-TestResource:

https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/TestResources/New-TestResources.ps1#L365-L367

https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/TestResources/New-TestResources.ps1#L436-L439

might be good to unify it in one place